package.json

```diff
@@ -55,6 +59,7 @@
     "classnames": "^2.3.1",
     "dayjs": "^1.10.7",
     "immer": "^9.0.1",
+    "moment": "^2.29.1",
```

Comment:
Why is it needed? We really don't want to re-add moment when we just got upstream to remove it

Reply:
It was used by me once, I changed it, see next comment!